internal/cli/admincmd/install_client.go

```diff
@@ -350,6 +315,13 @@ func (c *InstallClient) InstallIngress(cmd *cobra.Command) error {
 		return err
 	}
 
+	s := c.ui.Progressf("Waiting for LoadBalancer IP on traefik service.")
+	if err := c.waitForTraefikIngressIP(ctx); err != nil {
+		s.Stop()
+		return err
+	}
+	s.Stop()
```

Comment:
Seeing the entire codeblock above a second time, see also lines 169-174, place into a (local) helper function ?
That could then also use `defer s.Stop()` to simplify the progress handling a bit.

Reply:
Done in [7affb046](https://github.com/epinio/epinio/pull/983/commits/7affb0467816601b3433ea3f1904ecf4072d04f1)